smoke-test/tests/cypress/cypress/e2e/auto_complete/auto_complete.js

```diff
@@ -1,56 +1,64 @@
 
-        // look for a dataflow
-        cy.get("input[data-testid=search-input]").type("dataflow user");
-        cy.contains("Pipelines")
-        cy.contains("Users")
-        cy.focused().clear();
-    });
+    // look for a dashboard
+    cy.get("input[data-testid=search-input]").type("baz");
+    cy.contains("Dashboards");
+    cy.contains("Baz Dashboard");
+    cy.focused().clear();
 
-    it('should send you to the entity profile after clicking on an auto-complete option', () => {
-        cy.login();
-        cy.visit("/");
-        cy.get("input[data-testid=search-input]").type("SampleCypressHiveDataset");
-        cy.get('[data-testid="auto-complete-option"]').first().click();
-        cy.url().should('include', 'dataset/urn:li:dataset:(urn:li:dataPlatform:hive,SampleCypressHiveDataset,PROD)');
-        cy.wait(2000);
-    });
+    // look for a dataflow
+    cy.get("input[data-testid=search-input]").type("dataflow user");
+    cy.contains("Pipelines");
+    cy.contains("Users");
+    cy.focused().clear();
+  });
 
-    it('should filter auto-complete results when clicking on a quick filter', () => {
-        cy.login();
-        cy.visit("/");
-        cy.get("input[data-testid=search-input]").type("baz");
-        cy.contains("Baz Chart 2")
-        cy.get('[data-testid="quick-filter-DASHBOARD"]').click();
-        cy.wait(2000);
-        cy.ensureTextNotPresent("Baz Chart 2");
-        cy.contains("Baz Dashboard");
-        cy.wait(1000);
-    });
+  it("should send you to the entity profile after clicking on an auto-complete option", () => {
+    cy.login();
+    cy.visit("/");
+    cy.get("input[data-testid=search-input]").type("SampleCypressHiveDataset");
+    cy.get('[data-testid^="auto-complete-option"]').first().click();
+    cy.url().should(
+      "include",
+      "dataset/urn:li:dataset:(urn:li:dataPlatform:hive,SampleCypressHiveDataset,PROD)"
+    );
+    cy.wait(2000);
+  });
 
-    it('should filter search results when when searching with a quick filter selected', () => {
-        cy.login();
-        cy.visit("/");
-        cy.get("input[data-testid=search-input]").type("raw_pay");
-        cy.contains("raw_payments")
-        cy.get('[data-testid="quick-filter-urn:li:dataPlatform:bigquery"]').click();
-        cy.focused().type("{enter}");
-        cy.url().should('include', '?filter_platform___false___EQUAL___0=urn%3Ali%3AdataPlatform%3Abigquery');
-    });
-})
\ No newline at end of file
+  it.only("should filter auto-complete results when clicking on a quick filter", () => {
+    cy.login();
+    cy.visit("/");
+    cy.get("input[data-testid=search-input]").type("baz");
+
+    cy.get('[data-testid="auto-complete-option--urn:li:chart:(looker,baz2)"');
+    cy.get('[data-testid="quick-filter-DASHBOARD"]').click();
+    cy.wait(2000);
+    cy.get(
+      '[data-testid="auto-complete-option--urn:li:chart:(looker,baz2)"'
+    ).should("not.exist");
+    cy.contains("Baz Dashboard");
+    cy.wait(1000);
+  });
```

Comment:
The diff is hard to read... but the change here is is in the selectors in this test. Basically we attach urns to each data-testid in the autocomplete and makes selecting them simple. This way, we don't accidentally select "Baz Chart 2" which is sitting behind the autocomplete on a couple of links.